docs/tutorials/google/visualizing_calibration_metrics.ipynb

```diff
@@ -68,27 +68,29 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "9e4d953fcc28"
+    "cellView": "form",
+    "id": "T7hnERotRv12"
    },
    "outputs": [],
    "source": [
+    "#@title Necessary Imports\n",
     "try:\n",
     "    import cirq\n",
     "except ImportError:\n",
     "    print(\"installing cirq...\")\n",
-    "    !pip install --quiet cirq\n",
+    "    !pip install --quiet --pre cirq\n",
     "    print(\"installed cirq.\")\n",
     "    import cirq\n",
-    "\n",
     "import os\n",
     "from random import random\n",
-    "from itertools import combinations"
+    "from itertools import combinations\n",
+    "import matplotlib.pyplot as plt"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "89469d76dcf4"
+    "id": "4a2c0f8a990f"
    },
    "source": [
     "## Authenticate and install Cirq\n",
```

Comment:
I know this isn't part of the changes in this PR, but go-links like this should not be included in external documentation. (It's not a security issue, it's just Google policy.)

Reply:
done. Also sent #3963 to fix the root cause :)